README.md

```diff
@@ -1,12 +1,189 @@
-# gql-zero
+[![GitHub release](https://img.shields.io/github/tag/manifoldco/gql-zero.svg?label=latest)](https://github.com/manifoldco/gql-zero/releases) [![Travis](https://img.shields.io/travis/manifoldco/gql-zero/master.svg)](https://travis-ci.org/manifoldco/gql-zero) [![License](https://img.shields.io/badge/license-BSD-blue.svg)](./LICENSE.md)
 
-A lightweight alternative for consuming GraphQL. Zero caching, zero bells. Zero Bloat.
+# ⚡️ gql-zero
 
-[Code of Conduct](./CODE_OF_CONDUCT.md) |
-[Contribution Guidelines](./.github/CONTRIBUTING.md)
+A `0.4 KB` alternative for consuming GraphQL. Zero caching, zero bells. Zero
+Bloat.
 
-[![GitHub release](https://img.shields.io/github/tag/manifoldco/gql-zero.svg?label=latest)](https://github.com/manifoldco/gql-zero/releases)
+## Usage
 
-[![Travis](https://img.shields.io/travis/manifoldco/gql-zero/master.svg)](https://travis-ci.org/manifoldco/gql-zero)
+```bash
+npm i @manifoldco/gql-zero
+```
 
-[![License](https://img.shields.io/badge/license-BSD-blue.svg)](./LICENSE.md)
+### 🧁 Using Vanilla JS fetch
```

Comment:
I mean, it's vanilla TS. Maybe we should show both?

Reply:
Would it be clear to devs to just remove the types? Or no? I was thinking duplicating this just for a few types might add unnecessary lines to the readme. But I can maybe add “TS” somewhere